crates/swc_common/src/lib.rs

```diff
@@ -85,3 +87,9 @@ pub use self::syntax_pos::{
     ArchivedBytePos, ArchivedCharPos, ArchivedFileName, ArchivedMultiSpan, ArchivedSourceFile,
     ArchivedSourceFileAndBytePos, ArchivedSpan, ArchivedSpanLinesError, ArchivedSpanSnippetError,
 };
+
+#[cfg(swc_ast_unknown)]
+#[track_caller]
+pub fn unknown() -> ! {
```

Comment:
We cannot use `!` type because of the MSRV

Reply:
Okay, I'll change it to a macro.

Reply:
You can use Infailible